### PR TITLE
Added metadata to longrunning config.

### LINKED ISSUE #766: [LRO] Add metadata_type to longrunning field in gapic yaml.
### What
[google.longrunning.Operation](https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto#L80) objects hold a metadata google.protobuf.Any message. A type is needed in order for LRO wrappers to be able to unpack the metadata from the message. There is currently specification for the [response type](https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/library_gapic.yaml#L502) to unpack the response of an Operation but not the metadata. This needs to be added.

### Expected
```yaml
  - name: GetBigBook
    ...
    long_running:
      return_type: google.example.library.v1.Book
      metadata_type: google.example.library.v1.GetBigBookMetadata
```

### PR DESCRIPTION
**Pull request by landrito**
Resolves #766.

### CODE REVIEW

File: src/main/java/com/google/api/codegen/config/LongRunningConfig.java [hunk 1]
```diff
@@ -30,6 +30,9 @@
   /** Returns the message type returned from a completed operation. */
   public abstract TypeRef getReturnType();
 
+  /** Returns the message type returned from a completed operation. */
+  public abstract TypeRef getMetdataType();
```
Comment (garrettjonesgoogle) [inline src/main/java/com/google/api/codegen/config/LongRunningConfig.java]: fix spelling

File: src/main/proto/com/google/api/codegen/config.proto [hunk 2]
```diff
@@ -453,4 +453,7 @@ message LongRunningConfigProto {
   // The fully-qualified type that is returned from an Operation when it is
   // complete.
   string return_type = 1;
+
+  // The fully-qualified type  of the metadata of an Operation.
```
Comment (garrettjonesgoogle) [inline src/main/proto/com/google/api/codegen/config.proto]: convert the double space into a single space in the middle of the doc line here

File: src/main/java/com/google/api/codegen/config/LongRunningConfig.java (hunk 1 above)
Comment (landrito) [inline src/main/java/com/google/api/codegen/config/LongRunningConfig.java]: done.

File: src/main/proto/com/google/api/codegen/config.proto (hunk 2 above)
Comment (landrito) [inline src/main/proto/com/google/api/codegen/config.proto]: done.

File: src/main/java/com/google/api/codegen/config/LongRunningConfig.java (hunk 1 above)
Comment (landrito) [inline src/main/java/com/google/api/codegen/config/LongRunningConfig.java]: fixed!

**Pull request merged**